wails-frontend/src/pages/passwordPrompt.tsx

```diff
@@ -48,14 +48,11 @@ function PasswordPrompt() {
   const [error, setError] = useState<IErrorObject | null>(null);
 
   function validate(e: SyntheticEvent) {
-    const form = parseForm(e);
-    const { password } = form;
-
-    // reset error state
-    setError(null);
+    const formObject = parseForm(e);
+    const { password } = formObject;
 
-    if (!hasMoreThanFiveChars(password)) {
-      setError({ password: 'Password must have at least 5 characters' });
+    if (!password || !password.length) {
```

Comment:
Question: do we need both!?

Reply:
Feel free to close if is needed!

Reply:
variable returned by parseForm is type any, so yes